ccm_web/server/src/api/api.admin.handler.ts

```diff
@@ -103,4 +110,104 @@ export class AdminApiHandler {
     logger.debug(`Time elapsed: (${(end - start) / NS_PER_SEC}) seconds`)
     return finalResult
   }
+
+  async createExternalUser (user: SectionExternalUserDto, accountID: number): Promise<CanvasUser | APIErrorData> {
+    const email = user.email
+    const loginId = email.replace('@', '+')
+
+    try {
+      const endpoint = `accounts/${accountID}/users`
+      const method = HttpMethod.Post
+      const body = {
+        user: {
+          // API doesn't provide separate given- and surname fields
+          name: `${user.givenName} ${user.surname}`,
+          sortable_name: `${user.surname}, ${user.givenName}`,
+          terms_of_use: true,
```

Comment:
I **_think_** this will work without specifying the `terms…` key.  I could remove it, if you prefer.  I think I chose to use `true` here, because I read that's the default value Canvas users if it's not specified.

Reply:
Question for @melindakraft.

Reply:
Opened issue #291.

Reply:
Can we resolve #291 as part of this issue? It just needs a little testing I think. See my comment on that issue.

Reply:
Yes, I see your comment and @jonespm's.  I'll try to resolve that here.